wolfssl/test.h

```diff
@@ -1681,6 +1687,24 @@ static WC_INLINE int myVerify(int preverify, WOLFSSL_X509_STORE_CTX* store)
                                                                   subject);
         XFREE(subject, 0, DYNAMIC_TYPE_OPENSSL);
         XFREE(issuer,  0, DYNAMIC_TYPE_OPENSSL);
+#if defined(SHOW_CERTS) && !defined(NO_FILESYSTEM)
+/* avoid printing same certs since myVerify is called for every cert in the chain */
+        if (store->depth == 1) {
```

Comment:
I think this should be “error_depth“. Pretty sure this logic will only work if there is only 1 cert presented by peer. And then revert the “depth” changes.

Reply:
@dgarske, I tested the changes with two certs presented by peer and it works. error_depth won't work since we are interested in locating the last cert in the chain.